### ISSUE TITLE
Installing inside of Container has weird side-effect

### ISSUE THREAD

**Issue by metruzanca** [deepjyoti30/ytmdl#156]
# Bug report

## Checklist

- [x] I have searched for similar bugs in the issues.
- [x] I will add the bug label to this issue.

## Version Info

- Ytmdl version: `2021.01.14`
- Youtube-dl version: `N/A - Don't have it installed separately`

## How Ytmdl was installed?

I'm running it in docker, heres the docker file.
https://gist.github.com/metruzanca/31b2451747b8cce4fe5eda4e840cd410/648939aa72579befe89eb1c6f79780da7ce35c14

## Console Log

N/A

## Description

In my personal environment, I'm running it as a one shot type cli since I don't want to install stuff like python and ffmpeg on my docker machine so I've just made it into a lightweight docker image that I can use as its own executable cli.

> side note: I will probably be integrating this with my self-hosted music setup, definitely taking advantage of the from file feature to bulk download things from youtube mixes I like that usually have a tracklist in the description.

Its works fine with commands like `docker run --rm -it -v ~/Downloads:/downloads ytmdl "rob zombie Dragula"` however, when the file is done downloading I'm left with an empty ytmdl directory as a side-effect. I presume this is the folder for the config that gets generated, but it seems like it doesn't generate due to the fact that there is no USER in this container.

~~Odly enough passing --level DEBUG removes this side effect. So for the time being, I think I'll just pass that.~~ nvm this doesn't work

**Comment by deepjyoti30** [comment]
@metruzanca Have you set the download directory in the `config` file?

Since the newer versions, if the download directory is not specified than the file is downloaded to the current directory that the user is on. Also `ytmdl` works on the file in the `~/.cache/ytmdl` directory, you can take a look there to see if the files are still there. Also perhaps you can find the `log` in there as well.

**Comment by deepjyoti30** [comment]
@metruzanca If you're still interested, recently I had installed `ytmdl` in a Docker container as well and looking at the information you provided, here is an obsevation I made

`ytmdl` will by default download to the current directory (which should be `/app` in the case of Docker).

It seems like you're mounting your `~/Downloads` directory to the `/downloads` directory inside Docker. In that case, I think you'll need to add an `ytmdl config` that sets the song directory to `/downloads`.
